tests/attention/test_extreme_logits_regression.py

```
@@ -0,0 +1,303 @@
+        )
+
+        print(f"✓ Boundary test logit={target_logit:.0f} ({expected_behavior}): norm={output_norm:.4f}, rel_error={rel_error:.6f}")
+
+
+if __name__ == "__main__":
+    print("Running regression tests for issue #4267...")
+
+    try:
+        test_extreme_negative_logits_prefill_fp16()
+        print("✓ test_extreme_negative_logits_prefill_fp16 PASSED")
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)